src/content/write.rs

```diff
@@ -3,11 +3,9 @@ use std::io::prelude::*;
 use std::path::{Path, PathBuf};
 use std::pin::Pin;
 use std::sync::Mutex;
+use std::task::{Context, Poll};
 
-use async_std::fs as afs;
-use async_std::future::Future;
-use async_std::task::{self, Context, JoinHandle, Poll};
-use futures::io::AsyncWrite;
+use crate::async_lib::{AsyncWrite, JoinHandle};
```

Comment:
I prefer putting crate imports in their own group: https://github.com/zkat/cacache-rs/pull/36/files#diff-df142af2446d126a2fcd471722940667cc2a680e3496dc3f25acb067b3e17d53R8